api/src/pcapi/core/offerers/repository.py

```diff
@@ -162,7 +162,9 @@ def get_filtered_venues(
         if validated_offerer:
             query = query.filter(models.Offerer.isValidated)
         else:
-            query = query.filter(~models.Offerer.isValidated)  # type: ignore [operator] # pylint: disable=invalid-unary-operand-type
+            query = query.filter(models.Offerer.isWaitingForValidation)
+    else:
+        query = query.filter(sqla.not_(models.Offerer.isRejected))
```

Comment:
Est-ce qu'il y aurait moyen de simplifier ce gros bloc ?
Le premier enchaînement `if validated_offerer is not None` suivi d'un `if validated_offerer` est un peu étrange.

Peut-être quelque chose comme : 

```python
if validated_offerer is True:
    query = query.filter(models.Offerer.isValidated)
elif validated_offerer is False:
    query = query.filter(models.Offerer.isWaitingForValidation)
else:
    query = query.filter(sqla.not_(models.Offerer.isRejected))
```

Qu'en penses-tu ?
Et peut-être aussi renommer la variable `validated_offerer` en `is_offerer_validated` pour qu'on comprenne immédiatement que c'est un booléen ?

Reply:
C'est le code de l'équipe pro (dans le sens une API qui sert au portail pro), je ne vise pas à faire du _refactoring_ dedans, donc je mets à jour en conservant leur style de codage.